compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -745,9 +757,28 @@ goToDefinitionTests mbScenarioService = Tasty.testGroup "Go to definition tests"
             setFilesOfInterest [foo]
             expectNoErrors
             expectGoToDefinition (foo, 2, [7..14]) (In "DA.Internal.LF")
+
+    ,    testCase' "Exception goto definition" $ do
+            foo <- makeModule "Foo"
+                [ "exception Err where"
+                , "type E = Err"
+                ]
+            setFilesOfInterest [foo]
+            expectNoErrors
+            expectGoToDefinition (foo, 2, [9..11]) (At (foo,1,10))
+
+    ,    testCaseDev' "Interface goto definition" $ do
+            foo <- makeModule "Foo"
+                [ "interface Iface where"
```

Comment:
Interfaces generate three types of symbols for which go to definition should work:
1. Interface names
2. Choice names
3. Method names

You covered 1 here but we should also support 2 and 3. It might already work (haven’t checked) so could just be a matter of adding tests to prove that it works and continues working.